apps/comments-ui/src/actions.ts

```diff
@@ -23,18 +23,27 @@ async function loadMoreComments({state, api, options, order}: {state: EditableAp
 }
 
 async function setOrder({state, data: {order}, options, api}: {state: EditableAppContext, data: {order: string}, options: CommentsOptions, api: GhostApi}) {
-    let data;
+    state.commentsIsLoading = true;
```

Comment:
This feels like a React anti-pattern, should we be updating a context-provided state object directly?